app/src/main/java/com/android/sample/model/activity/ListActivitiesViewModel.kt

```diff
@@ -43,6 +53,21 @@ constructor(
     repository.init { viewModelScope.launch { getActivities() } }
   }
 
+  // Function to update filter state
+  fun updateFilterState(
+      price: Double?,
+      placesAvailable: Int?,
+      mindateTimestamp: Timestamp?,
+      acDuration: String?,
+      seeOnlyPRO: Boolean?
+  ) {
+    maxPrice = price ?: Double.MAX_VALUE
+    availablePlaces = placesAvailable
+    minDate = mindateTimestamp
+    duration = acDuration
+    onlyPRO = seeOnlyPRO ?: false
+  }
```

Comment:
sets the fields to specific values that will be depended on when filtering activities